outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "kafka_nodes" {
+  description = "Container Host IP addresses of Kafka instances"
+  value       = hsdp_container_host.kafka_connect.*.private_ip
+}
+
+output "kafka_name_nodes" {
```

Comment:
`kafka_connect_nodes_names` or something to that effect

Reply:
fixed